.github/workflows/ci.yaml

```
@@ -171,6 +175,18 @@ jobs:
 
+      - name: Cache playwright binaries
+        uses: actions/cache@v4
+        id: playwright-cache
+        with:
+          path: ~/.cache/ms-playwright
+          key: playwright-${{ steps.npm-install.outputs.PLAYWRIGHT_VERSION }}
+          restore-keys: playwright-
+
+      - name: Install playwright
+        run: npx playwright install chromium --with-deps
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)